.npmignore

```diff
@@ -0,0 +1,107 @@
+# Logs
```

Comment:
Meh. I don't need this anymore since I'm using `files` which is better and suggested by oclif anyway.